src/containers/profile/ProfileSections/AccountSettings/AccountSettingsSections/UpdatePassword.jsx

```diff
@@ -0,0 +1,224 @@
+import React from "react";
+import { connect } from "react-redux";
+import "./AccountSettingsSections.css";
+import {
+  updatePasswordAndEmail,
+  resetPassword,
+} from "../../../../../actions/authActions";
+import { Form, Container, Button, Col, Modal } from "react-bootstrap";
+
+const UpdatePassword = (props) => {
+  const { auth, authError, errMessage, errKey } = props;
+  const operations = [
+    ["Updating your password", "Your password has been updated successfully"],
```

Comment:
make these objects
```
{
   label: "Updating your password",
   successMessage: "..."
}
```

in general you want to avoid arrays of arrays because using [0] and [1] lacks clarity